app/views/components/ReturnHistoryTableView.scala.html

```diff
@@ -0,0 +1,57 @@
+@*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import viewmodels.ReturnHistorySummary
+
+@(rows: Seq[ReturnHistorySummary], colOne: String, colTwo: String, colThree: String, colFour: String, colFive: String, caption: String)(implicit messages: Messages)
```

Comment:
very minor: as this is a component the class name should probably just be ReturnHistoryTable

Also I take it there was a reason raw html was chosen over the GovUkTable component?

Reply:
Renamed components. Reason for raw html is that I based this on other project's component. This is temporary solution for now, if it stays we can refactor.